### PR TITLE
DPDV-3068 fix: config page for Alert triggered propagation splunk data to DataSet

### PR DESCRIPTION
**Pull request by zdaratom**
Jira Link: <https://sentinelone.atlassian.net/browse/DPDV-3068>

# 🥅 Goal

Fix Alert config page broken since https://sentinelone.atlassian.net/browse/DPDV-2604

# 🛠️ Solution

Allow user to specify parameters passed to DataSet while creating Splunk alert
added account and parse as described in https://github.com/scalyr/dataset-addon-for-splunk/blob/main/globalConfig.json#L650

# 🏫 Testing

Verified locally
screenshot-localhost_8000-2023 08 23-14_09_51

and [Prod DataSet](https://app.scalyr.com/events?teamToken=EhNwAjVvqmoNPU13tPbKbQ--&logSource=splunk&startTime=1692791608638&endTime=1692794656624)
screenshot-app scalyr com-2023 08 23-14_42_43

### CODE REVIEW

**Review by martin-majlis-s1 — COMMENTED** [review]
If you are able to find it in the Dataset, it should be possible to find in the Splunk itself.

Could you, please, try to extend the test scenario to really create that alert and then query it?

If it's not possible, could you, please, add it as comment to the test?

File: e2e/alert_action.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    console.log("Add trigger action - Dataset");
+    await page.getByText("+ Add Actions").click();
+    await page.getByText("Send to DataSet").click();
+    await page.screenshot({path: `playwright-screenshots/page-alert-dialog-dataset-trigger.png`, fullPage: true});
+    await expect(page.getByLabel(`ServerHost`)).toBeVisible();
+    await expect(page.getByLabel(`ServerHost`)).toBeVisible();
+    await expect(page.getByLabel(`DataSet Message`)).toBeVisible();
+    await expect(page.getByLabel(`Severity`)).toBeVisible();
+    await expect(page.getByLabel(`ServerHost`)).toBeVisible();
```
Comment (martin-majlis-s1) [inline e2e/alert_action.spec.ts:36]: It looks to me, that we are `await expect(page.getByLabel(`ServerHost`)).toBeVisible();` - maybe we should be checking different elements?

File: e2e/alert_action.spec.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    console.log("Fill in Alert Dialog - trigger DataSet event");
+    await setTimeout(500); // wait for select option to be populated
+    await page.locator('[name="action.dataset_event.param.account"]').locator('button').click();
+    await page.locator('[data-test="option"]').first().click(); // select first account
+    await page.locator('[name="action.dataset_event.param.dataset_serverhost"]').fill(serverHost)
```
Comment (martin-majlis-s1) [inline e2e/alert_action.spec.ts:42]: When we are filling the server host, we are imho implicitly checking, that it's there and is visible. At least this is my expectation, so those checks above are redundant?

**Review by martin-majlis-s1 — APPROVED** [review]

File: e2e/alert_action.spec.ts (hunk 1 above)
Comment (zdaratom) [reply e2e/alert_action.spec.ts]: I will remove redundant assetions

File: e2e/alert_action.spec.ts (hunk 2 above)
Comment (zdaratom) [reply e2e/alert_action.spec.ts:40]: true, I would prefer to keep that previous assertion so its consistent (verification that all required elements are there all in one place)

**Review by martin-majlis-s1 — APPROVED** [review]

**Pull request merged**